qa/tasks/ceph_manager.py

```diff
@@ -965,16 +965,16 @@ def test_pool_min_size(self):
                     for i in range(1, int(rand_val * len(self.dead_osds))):
                         self.revive_osd(i)
 
-            # let PGs repair themselves or our next knockout might kill one
-            self.ceph_manager.wait_for_clean(timeout=self.config.get('timeout'))
- 
-        # / while not self.stopping
-        self.all_up_in()
- 
-        self.ceph_manager.wait_for_recovery(
-            timeout=self.config.get('timeout')
+            self.ceph_manager.wait_for_recovery(
```

Comment:
Add a comment like
```
# let PGs repair themselves or our next knockout might kill one
# wait_for_recovery since some workloads won't be able to go clean
```